src/components/DashboardComponents/DashboardNews.js

```diff
@@ -0,0 +1,72 @@
+import { Box, Flex, HStack, Spacer} from "@chakra-ui/react"
```

Comment:
Please fix these two minor warnings when the project compiles:

```
[eslint]
src/components/DashboardComponents/DashboardNews.js
  Line 1:21:  'HStack' is defined but never used  no-unused-vars
  Line 1:29:  'Spacer' is defined but never used  no-unused-vars
```

Reply:
Done